content/applications/finance/accounting/taxation/fiscal_year/close_fiscal_year.rst

```diff
@@ -2,92 +2,91 @@
 Do a year end in Odoo (close a fiscal year)
 ===========================================
 
-Before going ahead with closing a fiscal year, there are a few steps one
-should typically take to ensure that your accounting is correct, up to
-date, and accurate:
+Before going ahead with closing a fiscal year, there are a few steps one should typically take to
+ensure that your accounting is correct, up-to-date, and accurate:
 
-- Make sure you have fully reconciled your **bank account(s)** up to
-  year end and confirm that your ending book balances agree with
-  your bank statement balances.
+-  Make sure you have fully reconciled your **bank account(s)** up to year-end and confirm that your
+   ending book balances match your bank statement balances.
 
--  Verify that all **customer invoices** have been entered and approved.
+-  Verify that all :guilabel:`Customer Invoice` have been entered and approved and that there are no
+   more draft invoices.
 
--  Confirm that you have entered and agreed all **vendor bills**.
+-  Confirm that you have entered and agreed on all :guilabel:`Vendor bills`.
 
--  Validate all **expenses**, ensuring their accuracy.
+-  Validate all :guilabel:`Expenses`, ensuring their accuracy.
 
-- Corroborate that all **received payments** have been entered and
-  recorded accurately.
+-  Corroborate that all :guilabel:`Received payments` have been encoded and recorded accurately.
+
+-  Close :guilabel:`Suspense accounts`.
+
+-  Book all the :guilabel:`Depreciation` and :guilabel:`Deferred Revenues`.
```

Comment:
general comments: add "see also" in this section or hyperlinks, to link to the related document pages?